set_environment.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
 export LICENSE_ID=
-export SPARCDIR=${VSC_DATA}/cryoSPARC
+export SPARCDIR=$VSC_SCRATCH_PROJECTS_BASE/apps/$VSC_INSTITUTE_CLUSTER/$VSC_ARCH_LOCAL$VSC_ARCH_SUFFIX/cryoSPARC
```

Comment:
The cryoSPARC master application is essentially a self-contained architecture-independent set of scripts. There is no real need to have a separate installation for each combination of `$VSC_INSTITUTE_CLUSTER/$VSC_ARCH_LOCAL$VSC_ARCH_SUFFIX`. In fact, since the installation contains a database, it is possible that users would want to share that database across different architectures, so perhaps one single installation is even preferable. I guess that this is not very important in practice at the moment, since I assume users will typically run the cryoSPARC master instance on a login node.

Reply:
I reverted in 45e8163